packages/contracts/bin/take-dump.ts

```diff
@@ -8,7 +8,7 @@ const CHAIN_ID = env.CHAIN_ID || '420'
 
 // Defaults to 0xFF....FF = *no upgrades*
 const L2_CHUG_SPLASH_DEPLOYER_OWNER = env.L2_CHUG_SPLASH_DEPLOYER_OWNER || '0x' + 'FF'.repeat(20)
-const CONGESTION_PRICE_ORACLE_OWNER = env.CONGESTION_PRICE_ORACLE_OWNER || '0x' + 'FF'.repeat(20)
+const GAS_PRICE_ORACLE_OWNER = env.GAS_PRICE_ORACLE_OWNER || '0x' + 'FF'.repeat(20)
```

Comment:
Calling this `gas price` might be confusing because we have an endpoint `eth_gasPrice` which will return a different value than what is in this contract. Because of that I think it makes sense to call this something else, whether it's congestion price or something else. What do you think?